tests/e2e-prometheuscr/create-sm-prometheus-exporters/06-assert.yaml

```diff
@@ -16,5 +16,6 @@ spec:
     - create-sm-prometheus
   selector:
     matchLabels:
-      app.kubernetes.io/managed-by: opentelemetry-operator
       app.kubernetes.io/instance: create-sm-prometheus.simplest
+      app.kubernetes.io/managed-by: opentelemetry-operator
```

Comment:
for these asserts, one thing that would also be helpful is to assert that the monitoring service for this selector has the labels the servicemonitor expects.